frontend/src/components/controls/RisNavbarSide.vue

```diff
@@ -140,7 +140,7 @@ export interface LevelTwoMenuItem {
   <aside aria-labelledby="sidebarNavigation" class="w-[16rem]">
     <span id="sidebarNavigation" hidden>SideBar Navigation</span>
     <router-link
-      aria-labelledby="toProceduresListButton"
+      aria-labelledby="toOverviewButton"
```

Comment:
Todo: IMHO this should be:

- Remove the `aria-labelledby`
- Remove the ID on the span that is supposed to be the label
- Leave the span there and add a [`sr-only`](https://tailwindcss.com/docs/screen-readers)

That should accomplish the same (button is labeled) without unnecessary use of aria or IDs, right?

I'm also wondering about the `aria-labelledby` of the aside, I don't think an aside needs a label in the first place and if it does it should probably be a hidden heading or something like that.

(I know you didn't implement this and only adjusted, but would still be grateful if you fixed it. I think this is an old-ish component and in those we sometimes have "unorthodox" accessibility stuff)

Reply:
I don't think that accomplishes the same thing as the text is currently visible for everyone. I'm not that knowledgeable about the aria stuff but i thought that this would ensure that the icon is not read out by the screenreader? Do you have another idea on how to solve this? Or is this already not a problem to begin with?

Reply:
i didn't adjust the aside to nav yet as i would first want to figure out if that has any other unexpected changes :)